tools/ci_build/github/azure-pipelines/templates/mac-packaging.yml

```diff
@@ -41,12 +45,22 @@ steps:
       displayName: 'Copy libcustom_op_library.dylib to ArtifactStagingDirectory'
       condition: and(succeeded(), eq('${{ parameters.MacosArch }}', 'x86_64'))
 
-    - ${{ if and(ne(parameters.MacosArch, 'universal2'), eq(parameters.buildJava, true)) }}:
+    - ${{ if eq(parameters.BuildJava, true) }}:
```

Comment:
Why do you remove the condition about universal2? I think java doesn’t need such a thing. It can load arm/x86 separately

Reply:
to make it consistent with "BuildNodejs". There is already a parameter to control whether run this job, and the parameter should be passed in when using this template.

Reply:
I see